src/index.js

```diff
@@ -1,43 +1,35 @@
-import AnchorSource from './components/Anchor'
-import AvatarSource from './components/Avatar'
 import ButtonSource from './components/Button'
-import CardSource from './components/Card'
 // import CategoriesBrowserImport from './components/CategoriesBrowser'
-// import FooterImport from './components/Footer'
-import CourseCardSource from './components/Card/CourseCard'
-import ErrorSource from './components/Error'
-import LessonCardSource from './components/Card/LessonCard'
-import PlaylistCardSource from './components/Card/PlaylistCard'
 import CheckboxSource from './components/Checkbox'
+import CourseCardSource from './components/CourseCard'
+import ErrorSource from './components/Error'
+// import FooterImport from './components/Footer'
 import HeadingSource from './components/Heading'
 import IconSource from './components/Icon'
 import InputSource from './components/Input'
-import LoadingSource from './components/Loading'
+import LessonCardSource from './components/LessonCard'
+import LogoSource from './components/Logo'
+import ParagraphSource from './components/Paragraph'
+// import PlaylistSource from './components/Playlist'
+import PlaylistCardSource from './components/PlaylistCard'
 import RadioGroupSource from './components/RadioGroup'
 import RequestSource from './components/Request'
-import SplitSource from './components/Split'
-import TabsSource from './components/Tabs'
 import TooltipSource from './components/Tooltip'
-import WellSource from './components/Well'
 
-export const Anchor = AnchorSource
-export const Avatar = AvatarSource
 export const Button = ButtonSource
-export const Card = CardSource
 // export const CategoriesBrowser = CategoriesBrowserImport
-// export const Footer = FooterImport
+export const Checkbox = CheckboxSource
 export const CourseCard = CourseCardSource
 export const Error = ErrorSource
-export const LessonCard = LessonCardSource
-export const PlaylistCard = PlaylistCardSource
-export const Checkbox = CheckboxSource
+// export const Footer = FooterImport
 export const Heading = HeadingSource
 export const Icon = IconSource
 export const Input = InputSource
-export const Loading = LoadingSource
+export const LessonCard = LessonCardSource
+export const Logo = LogoSource
+export const Paragraph = ParagraphSource
+// export const Playlist = PlaylistSource
+export const PlaylistCard = PlaylistCardSource
 export const RadioGroup = RadioGroupSource
 export const Request = RequestSource
-export const Split = SplitSource
-export const Tabs = TabsSource
 export const Tooltip = TooltipSource
-export const Well = WellSource
```

Comment:
I removed unused imports that were broken - when consuming in another project. Also put them in the same order as src/components and only exported those that are used externally, not internal components like `Card`.